src/views/schedule/components/EventSpacerCard.tsx

```diff
@@ -0,0 +1,43 @@
+import dayjs from "dayjs";
+import Card from "react-bootstrap/Card";
+
+import styles from "./EventCard.module.scss";
+
+interface EventSpacerCardProps {
+	now: Date;
+	title: string;
+	start: Date;
+	end: Date;
+	description: string;
+}
+
+function EventSpacerCard({
+	now,
+	title,
+	start,
+	end,
+	description,
+}: EventSpacerCardProps) {
+	const duration = dayjs(end).diff(dayjs(start), "hour", true);
+	const durationText = duration + " hours";
+
+	const over = now > end;
+	const minHeight = 80 * (duration + 1);
+
+	return (
+		<Card
+			className={"text-end mb-4 " + (over && styles.pastEventCard)}
+			bg="dark"
+			text="white"
+			style={{ minHeight }}
+		>
+			<Card.Body className="d-flex flex-column justify-content-center">
+				<Card.Title as="h4">{title}</Card.Title>
+				<Card.Text>{description}</Card.Text>
+				<footer className="text-white-50 text-end">{durationText}</footer>
+			</Card.Body>
+		</Card>
```

Comment:
It might just be me, but all of these are aligned to the right instead of the center:
![image](https://user-images.githubusercontent.com/28026520/216682306-142c28c8-d7ba-47d1-9edc-15a8837e90a9.png)

Reply:
That's part of the style we used since HackUCI 2021. Open to other suggestions though.